workers/unity/ProjectSettings/GraphicsSettings.asset

```diff
@@ -38,6 +38,7 @@ GraphicsSettings:
   - {fileID: 17000, guid: 0000000000000000f000000000000000, type: 0}
   - {fileID: 16000, guid: 0000000000000000f000000000000000, type: 0}
   - {fileID: 16001, guid: 0000000000000000f000000000000000, type: 0}
+  - {fileID: 16002, guid: 0000000000000000f000000000000000, type: 0}
```

Comment:
❓

Reply:
This might have happened if you swapped your platform target at some point